include/ppx/grfx/grfx_swapchain.h

```diff
@@ -146,6 +146,8 @@ class Swapchain
         grfx::Fence*     pFence,     // Wait fence
         uint32_t*        pImageIndex);
 
+    Result Wait(uint32_t);
```

Comment:
the input arg uint32_t is never used, so maybe better remove it?

Reply:
the name `Wait` is a bit confusing. User might be confused what is it waiting for, and if it is a cpu or gpu wait. maybe add some comments or rename it something like "XRCPUWaitSwapchainImage" ?